File: Java/opentraceability/src/main/java/opentraceability/queries/EPCISQueryInterfaceOptions.java
package opentraceability.queries;

import opentraceability.mappers.EPCISDataFormat;
import opentraceability.models.events.EPCISVersion;
import opentraceability.models.identifiers.*;
import java.net.URI;

public class EPCISQueryInterfaceOptions {
    public URI url = null;

    public EPCISVersion version = EPCISVersion.V2;

    public EPCISDataFormat format = EPCISDataFormat.JSON;

    public boolean enableStackTrace = true;
}

File: Java/OpenTraceability/src/main/opentraceability/queries/epcisqueryinterfaceoptions.kt
package queries
import mappers.EPCISDataFormat
import models.identifiers.*
import models.events.*
import java.net.URI
class EPCISQueryInterfaceOptions {
    var URL: URI? = null
    var Version: EPCISVersion = EPCISVersion.V2
    var Format: EPCISDataFormat = EPCISDataFormat.JSON
    var EnableStackTrace: Boolean = true
}
